traffic_ops/app/lib/API/DeliveryServiceMatches.pm

```diff
@@ -28,11 +29,19 @@ sub index {
 	my $format = $self->param("format") || "";
 
 	my $rs;
-	if ( &is_privileged($self) ) {
+	# TO the reviewer: Do we need to override the "is_priviledged" here byt the standard "ignore_ds_user_table" flag?
+	# What is the reason of the is_priv test - was someone just dussmissed the ds_tmuser table tests
+	if ( &is_privileged($self)) {
```

Comment:
I was trying to keep the functionality as is, and just add the tenancy tests.
Do you know why was it restricted to a privileged user in the first place?